worldedit-core/src/main/java/com/fastasyncworldedit/core/math/LocalBlockVectorSet.java

```diff
@@ -58,6 +58,48 @@ public LocalBlockVectorSet(int x, int y, int z) {
         this.set = new SparseBitSet();
     }
 
+    /**
+     * New wrapped LocalBlockVectorSet allowing "upgrading" to a {@link BlockVectorSet} as necessary
+     *
+     * @param set existing {@link LocalBlockVectorSet} to wrap
+     * @since TODO
+     */
+    public static BlockVector3Set wrap(LocalBlockVectorSet set) {
+        return new BlockVector3SetHolder(set);
+    }
+
+    /**
+     * New wrapped LocalBlockVectorSet allowing "upgrading" to a {@link BlockVectorSet} as necessary
+     *
+     * @since TODO
+     */
+    public static BlockVector3Set wrapped() {
+        return new BlockVector3SetHolder(new LocalBlockVectorSet());
+    }
+
+    /**
+     * New wrapped LocalBlockVectorSet with initial offset allowing "upgrading" to a {@link BlockVectorSet} as necessary
+     *
+     * @param x x offset
+     * @param z z offset
+     * @since TODO
+     */
+    public static BlockVector3Set wrapped(int x, int z) {
```

Comment:
The offset isn't used in any way, what's the intention here?

Reply:
Had meant to pass it onto the localbv3set